### ISSUE TITLE
"docker build ./"  run failed

### ISSUE THREAD

**Issue by beyond-infra** [laraedit/laraedit-docker#51]
Cannot add PPA: 'ppa:rwky/redis'.
Please check that the PPA name or format is correct.

**Comment by ibourgeois** [comment]
Yep. I have rebuilt this image several times since my last commit to this repo. When i get home, I will push up a working version that is more in line with the latest release of Laravel Homestead. Thanks!
